src/components/MagPage/MagPage.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'; // gimme React
+import classNames from 'classnames/bind'; // gotta name more than one class youngblood
+import PreviewRow from './PreviewRow'; // Imports the Mag Previews component
+import styles from './styles/magazinePage.css'; // show me my stylesheets
+
+let cx = classNames.bind(styles);
+
+class MagPage extends React.Component {
+    render() {
+        return (
+
+        	<div>
+            	<img className={cx("bkgImage")} src="/img/misc/bedroom.png" />
+            	<div className = {cx("headerText")}>
+	            	<h1> MAGAZINES </h1>
+	            	<h4> READ THE NEWEST SUMMER 16 MAG! <br/><br/><br/></h4>
+	            </div>
+            	<div className={classNames("container-fluid")}>
+            		<div className = "row">
```

Comment:
In the future, we may want to create this section using a for loop over an array of image paths when we start to have more, but I think this is cool for now.